graphql/schema/types/distro.graphql

```diff
@@ -5,6 +5,22 @@ enum DistroSettingsAccess {
   VIEW
 }
 
+"""
+CopyDistroInput is the input to the copyDistro mutation.
+It contains information about a distro to be duplicated.
+"""
+input CopyDistroInput {
+  newDistroId: String!
+  distroIdToCopy: String!
+}
+
+"""
+Return type representing whether a distro was created and any validation errors
+"""
+type NewDistroPayload {
+  newDistroId: String!
```

Comment:
Mohamed linked to [this article](https://www.apollographql.com/blog/graphql/basics/designing-graphql-mutations/#designing-the-mutation-payload) a while back that proposes always using objects for payloads. I think it's a good idea tbh because it facilitates future changes and backwards compatibility.